src/app/public/components/about/about.component.html

```diff
@@ -134,21 +140,22 @@ <h3>Klima Campus</h3>
 						Verbraucherschutz gefördert.
 					</p>
 				</div>
+				<div class="supporter-mobile"><img src="../assets/logos/Logo-BMUV.png" alt="BMUV logo" /></div>
 			</div>
 		</div>
 	</div>
 
 	<!-- Funded by -->
 	<div class="fund-style">
-		<div>
+		<div class="supporter-desktop">
 			<img src="../assets/logos/Logo-Telekom-Stiftung-2.jpg" alt="Telekom-Stiftung logo" />
 		</div>
 
-		<div>
-			<img style="width: 279px" src="../assets/logos/Logo-BMBF.png" alt="BMBF logo" />
+		<div class="supporter-desktop">
+			<img src="../assets/logos/Logo-BMBF.png" alt="BMBF logo" />
 		</div>
 
-		<div><img style="width: 442px" src="../assets/logos/Logo-BMUV.png" alt="BMUV logo" /></div>
+		<div class="supporter-desktop"><img style="width: 442px" src="../assets/logos/Logo-BMUV.png" alt="BMUV logo" /></div>
```

Comment:
I think the width should be either part of the ```supporter-desktop```class or applied as a tailwind class